addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1906,7 +1936,6 @@
 
                 input[type=range] {
                     pointer-events: none;
-                    focusable: false;
```

Comment:
Not needed anymore?

Reply:
I removed it because it is not a valid css property, maybe it was out of reach for this PR though